mne/proj.py

```diff
@@ -182,12 +182,18 @@ def compute_proj_epochs(
     ----------
     epochs : instance of Epochs
         The epochs containing the artifact.
-    n_grad : int
-        Number of vectors for gradiometers.
-    n_mag : int
-        Number of vectors for magnetometers.
-    n_eeg : int
-        Number of vectors for EEG channels.
+    n_grad : int | float between ``0`` and ``1``
+        Number of vectors for gradiometers. Either an integer or a float between 0 and 1
+        to select the number of vectors to explain the cumulative variance greater than
+        ``n_grad``.
```

Comment:
Should be a new `docdict` entry I think to make it DRY (we're gradually adding these as we come across them, so now's a good time for these params!)